CMakeLists.txt

```diff
@@ -114,6 +114,17 @@ install(
 set(SIGUTILS_UTIL_HEADERS ${UTILDIR}/util.h)
 set(SIGUTILS_UTIL_SOURCES ${UTILDIR}/util.c)
 
+if(WIN32)
+	set(SIGUTILS_UTIL_HEADERS ${SIGUTILS_UTIL_HEADERS} ${UTILDIR}/compat-mman.h)
```

Comment:
`compat-*.*` files should be compiled for all targets (as these are the ones that define macros conditionally). The `win32-*.*` are the per-target ones.